Ladybird/AppKit/UI/InspectorController.h

```diff
@@ -14,6 +14,5 @@
 @interface InspectorController : NSWindowController
 
 - (instancetype)init:(Tab*)tab;
-- (void)close:(BOOL)should_destroy_inspector;
```

Comment:
This commit also seems to just amend an earlier commit. Please squash these changes there.